R/mod_comments_table.R

```diff
@@ -46,10 +46,14 @@ mod_comments_table_server <- function(input, output, session, comments_promise)
   output$comments_table <- DT::renderDataTable({
     req(comments_promise())
     
-    comments_promise() %...>%
-      mutate(text = str_trunc(text, 80)) %...>%
-      select(time, by, sentiment) %...>% 
-      DT::datatable(selection = "single")
+    comments_promise() %...>% {
+        comments <- .
+        validate(need(!is.null(comments), "Unfortunately there are no comments for this story."))
+        comments %>% pull(text)
```

Comment:
`pull` returns a vector, not a data.frame. I guess the `mutate` function will problably complain about it ;)

Reply:
I think `select` would work better here :)

Reply:
And I see you use this code in 4 places. Maybe we could create a method in one place and call it?

Reply:
yea, moved empty comments validation to upper module